src/components/select/filterable-select/filterable-select.pw.tsx

```diff
@@ -720,7 +720,23 @@ test.describe("FilterableSelect component", () => {
     await expect(selectListWrapper(page)).toBeVisible();
   });
 
-  test("should open correct list and select one when an object is already set as a value", async ({
+  test("should have correct option highlighted when select list is opened and value is an object", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<FilterableSelectObjectAsValueComponent />);
+
+    const inputElement = getDataElementByValue(page, "input");
+    await expect(inputElement).toHaveValue("Green");
+    await dropdownButton(page).click();
+    const optionElement = selectOptionByText(page, "Green");
+    await expect(optionElement).toHaveCSS(
```

Comment:
**nitpick:** We've been putting a line break between the body of the test and the `expect()`.

```suggestion
    const optionElement = selectOptionByText(page, "Green");
    
    await expect(optionElement).toHaveCSS(
```